pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -318,14 +320,15 @@ func validateStaticStatus(ingressController operatorv1.IngressController, desire
 	if !(desiredSpec.Domain == ingressController.Status.Domain) {
 		return false
 	}
-
+	if !baseutils.IsVersionHigherThan("4.10") {
```

Comment:
(Optional, an idea): Instead of using checks in `validateStaticSpec()` and `validateStaticStatus()`, would it make sense to use a single check on https://github.com/openshift/cloud-ingress-operator/blob/master/pkg/controller/publishingstrategy/publishingstrategy_controller.go#L454 with something like the following to cut across the process in reconciliation

```go
if baseutils.IsVersionHigherThan("4.10")  {
	// Return and don't requeue
	return reconcile.Result{}, nil
}

Reply:
please ignore, it looks like mine was a wrong statement as it doesn't cover all the cases under `validateStaticSpec()` and `validateStaticStatus()`